### DISCUSSION TITLE
Default rank values?

### DISCUSSION THREAD

**Discussion by Tptuaazn** [BG-Software-LLC/SuperiorSkyblock2#218] [Denied Feature Requests] [answered]
![IMG_20210417_040717](https://user-images.githubusercontent.com/74866833/115084005-741a6480-9f32-11eb-94dc-92dc5ed56796.jpg)

**Comment by OmerBenGera** [comment, marked as answer]
Not possible as these values are per island and not per player, and can be easily exploited if it will be based on perms / ranks.
